src/common/mod.rs

```diff
@@ -0,0 +1,38 @@
+//! Common helpers for `displayz`
+
+/// Height and Width of a Display (`usize`)
+pub type Resolution = (usize, usize);
+
+/// X/Y positions of a display.
+pub type Position = (usize, usize);
+
+/// `Vec` type of the `Display` struct, exposed on a platform-dependent basis.
+pub type Displays = Vec<crate::Display>;
+
+/// `Vec` type of the `Resolution` type, generally exposing a collection of available resolutions.
+pub type Resolutions = Vec<Resolution>;
```

Comment:
not sure about this abstraction. I think Vec<Resolution> is more direct/clear

Reply:
Both abstractions or just one of them? The reason I did this was for a more implicit user-facing API, and making it user-friendly.

Reply:
I think both. Otherwise we obfuscate the fact that it's just a normal vec

Reply:
I wouldn't say we're obfuscating it, to be honest - it's documented in the rustdoc, and also, the IDE/editor of the developer using the library should illustrate it's a `Vec`. It's up to you if you want me to remove the abstraction, but I wanted to make it more developer-friendly by type aliasing the `Vec`.

Reply:
BTW, with `pub type Displays = Vec<crate::Display>;` - the idea I had behind that was that when the library is built on each 'platform', it uses the exported `Display` struct/impl from the platform. I might not be explaining that coherently - it's late here.

Reply:
Gotcha. Still not the biggest fan of `pub type Resolutions = Vec<Resolution>;`. But if you think that it improves the developer experience, feel free to keep it

Reply:
It's good to discuss these things. I'm just thinking from the developer side of things - but I've taken your views into account, and I'm going to add a comment to the Rustdoc of that type, to explain that it could be removed or refactored in future releases, depending on developer feedback.

Reply:
See 28620ba.

Reply:
I am not a big fan of this note either tbh. Seems like we have no confidence in our code. Either keep the alias or remove it.

Reply:
OK, I'll remove it. I'd rather keep the alias, but I don't really see any harm of adding a 'future change' Rustdoc.

Reply:
Reverted in 657ce27.